HUD/SetMinimapComponent.md

```diff
@@ -9,14 +9,26 @@ Any SET_MINIMAP_COMPONENT(int componentID, BOOL toggle, int hudColor);
 ```
 
 This native is used to colorize certain map components like the army base at the top of the map.
+Colors are being mixed, so for example using blue (HudColor: 9) on yellow lift-off markers
+will result in green instead of blue, because the base color is yellow.
```

Comment:
You marked this as resolved but didn't remove the new like between `markers` and `will`, letting the browser handle new lines will work better then adding them yourself